<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableView" name="tableView"/>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>分数段</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">数学</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>语文</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>英语</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>平均分</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>60分以下</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60～69分</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>70～79分</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80～89分</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>90分以上</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>平均分</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最高分</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最低分</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>BarChart</string>
      </attribute>
      <widget class="QChartView" name="chartViewBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>701</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuildBarChart">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新柱状图</string>
       </property>
      </widget>
      <zorder>btnBuildBarChart</zorder>
      <zorder>chartViewBar</zorder>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>PieChart</string>
      </attribute>
      <widget class="QChartView" name="chartViewPie">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>691</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>691</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="6">
         <widget class="QDoubleSpinBox" name="spinPieSize">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.700000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>HoleSize</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnPie">
          <property name="text">
           <string>刷新饼图</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cBoxCourse">
          <item>
           <property name="text">
            <string>语文</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>数学</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英语</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>平均分</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>分析数据</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>PieSize</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDoubleSpinBox" name="spinHoleSize">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>主题</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Light</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>StackedChart</string>
      </attribute>
      <widget class="QChartView" name="chartViewStackedBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>711</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStackedBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新堆叠柱状图</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab4">
      <attribute name="title">
       <string>PercentCharChart</string>
      </attribute>
      <widget class="QChartView" name="chartViewPercentBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>701</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPercentBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新百分比柱状图</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab5">
      <attribute name="title">
       <string>ScatterChart</string>
      </attribute>
      <widget class="QChartView" name="chartViewScatter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>701</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnScatter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新散点图</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actIni"/>
   <addaction name="actSurvey"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actIni">
   <property name="text">
    <string>初始化数据</string>
   </property>
   <property name="toolTip">
    <string>初始化数据</string>
   </property>
  </action>
  <action name="actSurvey">
   <property name="text">
    <string>数据统计</string>
   </property>
   <property name="toolTip">
    <string>数据统计</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
